### ISSUE TITLE
add index page

### ISSUE THREAD

**Issue by szabgab** [Abigail/Abigail.github.io#2]
If would be nice if each series had a list of the pages e.g. https://abigail.github.io/HTML/AdventOfCode/2021/ would list

https://abigail.github.io/HTML/AdventOfCode/2021/day-01.html

https://abigail.github.io/HTML/AdventOfCode/2021/day-02.html

etc.

**Comment by Abigail** [comment]
You mean, https://abigail.github.io/HTML/AdventOfCode/2021/index.html ? Which you can get to by type `i` on each of the `day-NN.html` pages, or by clicking the image in the top-right corner.

Not sure why github doesn't use `index.html` as the default page when querying a directory.

**Comment by szabgab** [comment]
yeah, that's what I meant. I wanted to link to it from the PW. Well, next week.
